specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,392 @@
+      "properties": {
+        "id": {
+          "type": "integer",
+          "description": "Id for the incident.",
+          "format": "int64"
+        },
+        "severity": {
+          "type": "integer",
+          "description": "Incident severity.",
+          "format": "int32"
+        },
+        "state": {
+          "type": "string",
+          "description": "Incident state."
+        },
+        "createdDate": {
+          "type": "string",
+          "description": "Incident created date."
+        },
+        "sourceCreateTime": {
+          "type": "string",
+          "description": "The date on which source created incident."
+        },
+        "lastModifiedDate": {
+          "type": "string",
+          "description": "Incident last modified date."
+        },
+        "title": {
+          "type": "string",
+          "description": "Title for the incident."
+        },
+        "summary": {
+          "type": "string",
+          "description": "Summary for the incident."
+        },
+        "description": {
+          "type": "string",
+          "description": "Description for the incident."
+        },
+        "cloudInstanceId": {
+          "type": "integer",
+          "description": "CloudInstanceId or TenantSiloId for the incident 's tenant.",
+          "format": "int64"
+        },
+        "responsibleServiceId": {
+          "type": "integer",
+          "description": "Service responsible for the incident.",
+          "format": "int64"
+        },
+        "responsibleTeamId": {
+          "type": "integer",
+          "description": "Team responsible for the incident.",
+          "format": "int64"
+        },
+        "owningServiceId": {
+          "type": "integer",
+          "description": "Owning service identifier for the incident.",
+          "format": "int64"
+        },
+        "serviceCategoryId": {
+          "type": "integer",
+          "description": "Id of the service category of the service that owns this incident.",
+          "format": "int64"
+        },
+        "owningTeamId": {
+          "type": "integer",
+          "description": "Owning team identifier for the incident.",
+          "format": "int64"
+        },
+        "originatingServiceId": {
+          "type": "integer",
+          "description": "Originating service identifier for the incident.",
+          "format": "int64"
+        },
+        "isCustomerImpacting": {
+          "type": "boolean",
+          "description": "A value indicating whether this instance is customer impacting."
+        },
+        "isNoise": {
+          "type": "boolean",
+          "description": "A value indicating whether this instance is noise."
+        },
+        "isSecurityRisk": {
+          "type": "boolean",
+          "description": "A value indicating whether this instance is security risk."
+        },
+        "keywords": {
+          "type": "string",
+          "description": "Keywords for the incident."
```

Comment:
Is this a comma separated or space separated string, might be worth including the format.

Reply:
that's a good point, i will add the format as soon as i get a more set answer on the format from the data source. I'd like to keep it as it is for now if possible.